docs/source/release_notes.rst

```diff
@@ -2,8 +2,10 @@ Release Notes
 -------------
 **Future Releases**
     * Enhancements
+        * Added ``inverse_transform`` method to pipelines :pr:`2256``
     * Fixes
     * Changes
+        * Updated logging information in ``AutoMLSearch.__init__`` to clarify pipeline generation :pr:`2263`
```

Comment:
@ParthivNaresh I'm moving this from the 0.24.1 release to the future release!

Reply:
@freddyaboulton Sure thing!